docs/Backup & restore.md

```diff
@@ -1,5 +1,7 @@
 # Backup & restore
 
+Supporting backup & restore is a key feature of IS-14. In addition to supporting full back and full restore to the same device, IS-14 enables partial backup and partial restore to the same device or a different device. To discuss the various possible combinations of backup and restore, this section utilizes terms explained in the [Definitions](#definitions) section.
```

Comment:
I feel we need to define more rigorously the terms "partial backup" and "partial restore".  

For instance there could be confusion over whether, when you take a partial backup and restore all of it to a Device Model, whether that is a full restore of a partial backup, or whether it's a partial restore of a partial backup.  And does the partial restore definition apply if we can only restore parts of the full backup etc.

Perhaps adding something in the Definitions section to stop that sort of confusion?